### ISSUE TITLE
Requires Statement.py and realtime_model_POOL_driver.py

### ISSUE THREAD

**Issue by trung6689** [Leci37/stocks-prediction-Machine-learning-RealTime-telegram#13]
Statement.py
realtime_model_POOL_driver.py
I'm very excited about this stock prediction project! The features that this project brings are truly special and appealing. The project has conducted experiments with not only 36 different models but also provides multiple options for combining features. This allows for easy scalability with technologies like TensorFlow, XGBoost, Sklearn, LSTM, GRU, Dense, LINEAR, and many other models. I can't resist my curiosity to explore more details about this project and the potential it holds. It's possible that this is a breakthrough in stock prediction. When carrying out the project, I found the need for the two files mentioned above. Can you provide me with full access to the project?

**Comment by Leci37** [comment]
Thank you , you are so gentle

For recive this files , It is only for colaborator on developer , you can join here  Telegram GROUP: https://t.me/+3oG6U_hp93I2M2Ix (response in less than one day )

I will be asked for training, what improvements you can think of. (recommended to review the point: https://github.com/Leci37/stocks-prediction-Machine-learning-RealTime-telegram/edit/master/README.md#possible-improvements).

**Comment by trung6689** [comment]
Hi @Leci37 
I have requested to join the Telegram GROUP. Could you please help me with that?

When running your source code, I also noticed some errors that might be due to Python library incompatibility.
I think improving the collection of stock news data will enhance the accuracy of the predictions.

**Comment by Leci37** [comment]
could you Give me the error stacks , if you solved it , how ?? i want to solve in the main branch

**Comment by trung6689** [comment]
When running the program, I noticed that there are some module requirements. After installing them, I didn't find anything missing. However, when I reached step 5, which requires the "realtime_model_POOL_driver.py" file, I want to continue running the program until the end. Is there any way to proceed, or is the project only shareable up to this point?

**Comment by Leci37** [comment]
I wrote you at telegram , maybe you block me 

you are no able to recive my message
